projects/admin/src/app/record/detail-view/library-detail-view/location/location.component.html

```diff
@@ -18,9 +18,10 @@
   <div class="col">
     <a [routerLink]="['/records/locations/detail', location.metadata.pid]">{{ location.metadata.name }}</a>
   </div>
-  <div class="col-1">
-    <i class="fa fa-truck ml-2" *ngIf="location.metadata.is_pickup" title="{{ 'Pickup location' | translate }}"></i>
-    <i class="fa fa-globe ml-1" *ngIf="location.metadata.is_online" title="{{ 'Online location' | translate }}"></i>
+  <div class="col-2 text-right">
+    <i class="fa fa-truck ml-2 text-primary" *ngIf="location.metadata.is_pickup" title="{{ 'Pickup location' | translate }}"></i>
+    <i class="fa fa-truck ml-2 text-warning" *ngIf="location.metadata.is_ill_pickup" title="{{ 'ILL pickup location' | translate }}"></i>
+    <i class="fa fa-globe ml-1 text-success" *ngIf="location.metadata.is_online" title="{{ 'Online location' | translate }}"></i>
```

Comment:
Blue, orange and green... it's already christmas !
I will let pickup and online in normal mode (black) and use blue (primary) for ill pickup.
Needs @lauren-d and other UI/UX specialist review :-)